pages/edge/tutorials/cdn-on-wasmer-edge.mdx

```diff
@@ -0,0 +1,357 @@
+  package. More info about it can be found
+  [here](https://static-web-server.net/configuration/config-file/).
+</Callout>
+
+Now, let's add the following configuration to the `wasmer.toml` file.
+
+```yaml
+[package]
+name = "<your-namespace>/<your-cdn-name>"
+version = "0.1.0"
+description = "<your-namespace>/<your-cdn-name> CDN on Wasmer Edge"
+
+[dependencies]
+"wasmer/static-web-server" = "^1"
+
+[fs]
+public = "public"
+config = "config" # 👈🏼 Add the config directory
+
+[[command]]
+name = "script"
+module = "wasmer/static-web-server-async:webserver"
+runner = "https://webc.org/runner/wasi"
+
+[command.annotations.wasi] # 👈🏼 Add config.toml as an argument for the webserver
+main-args = ["-w", "./config/config.toml"]
+```
+
+### Adding files to the CDN
+
+Now, let's add some files to the `public` directory.
+
+```bash
+$ touch public/index.html
+```
+
+I'll also add an image called `image.jpg` to the `public` directory.
+
+And now my `index.html` file looks like this:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <img src="./image.jpg" width="600" />
+  </body>
+</html>
+```
+
+You can also add other files like css, js, videos, pdfs, etc.
+
+You can view my app [here](https://dynamite-bud-sws-cdn.wasmer.app).
+
+### Deploying the CDN
+
+Now, let's deploy the CDN.
+
+```bash
+$ wasmer deploy
+```
+
+The above command will prompt you with the following:
+
+```bash
+Loaded app from: /path/to/your/dir/app.yaml
+
+Publish new version of package '<your-namespace>/<your-cdn-name>'? yes
+Publishing package...
+[1/2] ⬆️   Uploading...
+[2/2] 📦  Publishing...
+Successfully published package `<your-namespace>/<your-cdn-name>@0.1.0`
+Waiting for package to become available.....
+Package '<your-namespace>/<your-cdn-name>@0.1.0' published successfully!
+
+Deploying app <your-namespace>-<your-cdn-name>...
+
+ ✅ App <your-namespace>/<your-namespace>-<your-cdn-name> was successfully deployed!
+
+> App URL: https://dynamite-bud-sws-cdn.wasmer.app
+> Versioned URL: https://2qjh9iec55z4.id.wasmer.app
+> Admin dashboard: https://wasmer.io/apps/dynamite-bud-sws-cdn
+
+Waiting for new deployment to become available...
+(You can safely stop waiting now with CTRL-C)
+.
+New version is now reachable at https://dynamite-bud-sws-cdn.wasmer.app
+Deployment complete
+```
+
+<Callout type="info" emoji="ℹ️">
+  And now you can view your CDN at the URL mentioned above. You can also view
+  the admin dashboard of your CDN by visiting the URL mentioned above.
+</Callout>
+
+<Callout type="info" emoji="🌐">
+  You'll need to do `wasmer deploy` again if you add any new files to the
+  `public` directory.
+</Callout>
+
+<Callout type="warning" emoji="🚧">
+  The url of your CDN will be different from the one mentioned above.
+</Callout>
```

Comment:
There's not a "Your CDN". You don't build a CDN, you publish INTO a CDN. This needs rephrasing

Reply:
Rephrased to: `The url of your app will be different from the one mentioned above.`